### ISSUE TITLE
getPageTimeout is not working.

### ISSUE THREAD

**Issue by amodmahajan2016** [angular/protractor#5178]
getPageTimeout is waiting time for page to be loaded. Default wait time in 10 sec. I tried to change it to 1 second just for checking its functionality. I am expecting to give Error: Timed out waiting for page to load after 1000ms but it didn't. 
Conf file:
`exports.config = {
  
    specs: ['../Spec Files/DropdownHandling.js'],

    getPageTimeout: 1,

  
}`

Spec file:
`describe('Handling dropdown', () => {
    it('Verify options', () => {
        browser.get("https://www.mobikwik.com/");
        element(by.model('operator')).click();
});

    

});`

Even I tried by passing it to get(url, time) as well. But it didn't work.

Is it something like we can not set less than default page load time out

Looking forward for the help.

**Comment by AyeshaBegam** [comment]
Even i have the same issue it is not picking the elements and entering the data in to them.Could anyone please look in to this and give us the solution for this.

**Comment by ashishbansali** [comment]
What's the error ? Share the HTML element. Is the page angular ?

**Comment by AyeshaBegam** [comment]
Yes the page is angular and it is a mobile application. Error is: Failed: script timeout: result was not received in 11 seconds. It is not entering the data in elements.It used to work earlier. I have upgraded protractor to Version 5.4.2

 Login to application should login with Credentials given
  Message:
[31m    Failed: script timeout: result was not received in 11 seconds
      (Session info: chrome=72.0.3626.121)
      (Driver info: chromedriver=2.45.615291 (ec3682e3c9061c10f26ea9e5cdcf3c53f3f74387),platform=Windows NT 10.0.17134 x86_64)[0m
  Stack:
    ScriptTimeoutError: script timeout: result was not received in 11 seconds
      (Session info: chrome=72.0.3626.121)
      (Driver info: chromedriver=2.45.615291 (ec3682e3c9061c10f26ea9e5cdcf3c53f3f74387),platform=Windows NT 10.0.17134 x86_64)

**Comment by amodmahajan2016** [comment]
ayeshaBegam- your issue is different. You need to increase timeout for angular loading. Refer timeouta section of projtractor official page.

**Comment by AyeshaBegam** [comment]
Hi amod,
  Did that as well. Increased the timeout value but still has the
same problem.

**Comment by cnishina** [comment]
When logging in, is this an Angular page? Did you navigate to a non-angular page? Also browser.get only does a wait for the url to change. https://github.com/angular/protractor/blob/master/lib/browser.ts#L747

**Comment by Nossarian** [comment]
@AyeshaBegam was this issue ever resolved? I believe I'm also running into this.
